dev/Dockerfile

```diff
@@ -34,6 +34,10 @@ RUN yum -y install epel-release && \
     alternatives --install /usr/bin/mariadb_config mariadb_config /mariadb-connector-c-3.1.13-centos7-amd64/bin/mariadb_config 1 && \
     alternatives --install /usr/lib64/libmariadb.so.3 libmariadb.so.3 /mariadb-connector-c-3.1.13-centos7-amd64/lib/mariadb/libmariadb.so.3 1 && \
     alternatives --install /usr/lib64/libmariadb.so libmariadb.so /mariadb-connector-c-3.1.13-centos7-amd64/lib/mariadb/libmariadb.so.3 1 && \
+    yum -y install npm nodejs && \
+    npm install --global pyright && \
+    npm cache clean -f && \
+    yum -y remove --setopt=clean_requirements_on_remove=1 npm && \
```

Comment:
Any particular reason why you remove it again? I think it could add some value (e.g. installing/testing a new version) while also freeing so much space (`nodejs` should be the big guy here I guess).

Reply:
Just because it currently isn't needed for anything else. But yes, it is only 8MB (compared to ~60-70 for nodejs), so won't make a big difference.

Reply:
Note to reviewer: I changed it to keep `npm` installed.